p2p/dandelion-tower/src/pool/manager.rs

```diff
@@ -280,11 +278,13 @@ where
                     };
 
                     if let Err(e) = self.handle_incoming_tx(tx, routing_state, tx_id).await {
+                        #[expect(clippy::let_underscore_must_use, reason = "TODO")]
                         let _ = res_tx.send(());
 
                         tracing::error!("Error handling transaction in dandelion pool: {e}");
                         return;
                     }
+                    #[expect(clippy::let_underscore_must_use, reason = "TODO")]
                     let _ = res_tx.send(());
```

Comment:
What is the reason channel errors can be ignored here?

Reply:
An error just means the requestor dropped the response channel, it doesn't effect the behaviour of the service.